service_container/factories.rst

```diff
@@ -100,6 +100,82 @@ create its object:
     the configured class name may be used by compiler passes and therefore
     should be set to a sensible value.
 
+You can also use the ``factory`` option to create services from static methods
```

Comment:
`When the static factory method is on the same class as the created instance, the class name can be omitted from the factory declaration.`